JASP-Engine/JASP/R/descriptives.R

```diff
@@ -1427,4 +1437,82 @@ Descriptives <- function(jaspResults, dataset, options) {
     return(TRUE)
   else
     return(FALSE)
-}
\ No newline at end of file
+}
+
+.XbarchartNoId <- function(jaspResults, dataset, options) {
+
+  ready <- (length(options$variables) > 1)
+  if (!ready)
+    return()
+
+  jaspResults[["XbarPlot"]] <- createJaspPlot(title = "X bar chart", width = 650, aspectRatio = 1)
+  jaspResults[["XbarPlot"]]$dependOn(c("Xbarchart", "variables"))
+  jaspResults[["XbarPlot"]]$position <- 11
+  XbarPlot <- jaspResults[["XbarPlot"]]
+
+  data1= na.omit(as.data.frame(dataset))
+  means <- rowMeans(data1)
+  subgroups <- 1:length(means)
+  data2 <- data.frame(subgroups = subgroups, means = means)
+  sixsigma <- qcc::qcc(data1, type = 'xbar', plot = F)
```

Comment:
```suggestion
  sixsigma <- qcc::qcc(data1, type = 'xbar', plot = FALSE)
```
it's possible to do `F <- TRUE`, but not `FALSE <- TRUE`, so this is safer.